projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 ### This is a list detailing changes for all Jetpack releases.
 
+## 10.8-a.9 - 2022-03-23
+### Enhancements
+- External Media: add Openverse as an external Media provider.
+- Jetpack: open the QR Post feature to public, and improve its logo sizing.
+- Payment Block: clicking on payment links from email or WordPress.com Notifications/Reader will correctly open the corresponding payment form.
+- Search: show search benefits on the Jetpack disconnection screen.
+
+### Improved compatibility
+- Connection: Jetpack now relies on the Connection package webhooks for 'authorize' and 'authorize_redirect' actions.
```

Comment:
I don't think end-users will really be interested in / understand this. I think it could be in Other.